galette/templates/default/pages/preferences.html.twig

```diff
@@ -319,7 +319,7 @@
                 <i class="eye outline icon"></i>
                 {{ __('Public pages') }}
             </div>
-            <div class="ui stackable two column grid">
+            <div class="ui stackable three column grid">
```

Comment:
I've not tested, but public relarted pages in preferences will be reworked #603; maybe would it be better to work directly with reworked page and make PR against `feature/staff-gallery` branch?

Reply:
> I've not tested, but public relarted pages in preferences will be reworked #603; maybe would it be better to work directly with reworked page and make PR against `feature/staff-gallery` branch?

All right ! :+1: